exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillResultSetImpl.java

```diff
@@ -1384,7 +1402,7 @@ public void updateRowId( String columnLabel, RowId x ) throws SQLException {
 
   @Override
   public int getHoldability() throws SQLException {
-    throwIfClosed();
+    throwIfTimedOutOrClosed();
```

Comment:
not sure if it makes sense (and correct per spec) to throw SqlTimeoutException here... (and similar methods)